<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BurnTools Ver 1.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/images/easyicon_net.png</normaloff>:/images/easyicon_net.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>761</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>输出信息</string>
    </property>
    <widget class="QTextEdit" name="textEdit_outputLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>721</width>
       <height>141</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>761</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>固件下载</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>当前固件类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>固件版本号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_firmwareType">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_firmwareVersion">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scanNode">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>选择设备</string>
     </property>
    </widget>
    <widget class="QPushButton" name="executeFirmwarePushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>执行固件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateFirmwarePushButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>更新固件</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="firmwareLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="openFirmwareFilePushButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>操作CAN</string>
    </property>
    <widget class="QComboBox" name="comboBox_selectCan">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>28</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>选择CAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CAN1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CAN2</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_closeCan">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>关闭CAN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_openCan">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>打开CAN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFirmwareFileAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="scanNodeAction"/>
    <addaction name="clearDisplayAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFirmwareFileAction"/>
   <addaction name="scanNodeAction"/>
   <addaction name="clearDisplayAction"/>
   <addaction name="aboutAction"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/gif_48_026.gif</normaloff>:/images/gif_48_026.gif</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="scanNodeAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/gif_48_086.gif</normaloff>:/images/gif_48_086.gif</iconset>
   </property>
   <property name="text">
    <string>选择设备</string>
   </property>
  </action>
  <action name="clearDisplayAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/gif_48_082.gif</normaloff>:/images/gif_48_082.gif</iconset>
   </property>
   <property name="text">
    <string>清除显示</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/gif_48_052.gif</normaloff>:/images/gif_48_052.gif</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="openFirmwareFileAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/images/gif_48_045.gif</normaloff>:/images/gif_48_045.gif</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
